packages/openneuro-components/src/mock-content/front-page-content.jsx

```diff
@@ -25,9 +26,9 @@ export const frontPage = {
   pageDescription: (
     <h2>
       A free and open platform for validating and sharing BIDS-compliant{' '}
-      <a href="/search/mri">MRI</a>, <a href="/search/pet">PET</a>,{' '}
-      <a href="/search/meg">MEG</a>, <a href="/search/eeg">EEG</a>, and{' '}
-      <a href="/search/asl">iEEG</a> data
+      <Link to="/search/mri">MRI</Link>, <Link to="/search/pet">PET</Link>,{' '}
+      <Link to="/search/meg">MEG</Link>, <Link to="/search/eeg">EEG</Link>, and{' '}
+      <Link to="/search/asl">iEEG</Link> data
```

Comment:
These should use the full URL instead of the redirect, `/search/modality/eeg` to avoid search engines confusing the canonical version.

Reply:
I'll handle the empty keyword chiclet and x-ing out the modality along with the change requests to #2179.